CommunityToolkit.Mvvm/ComponentModel/ObservableValidator.cs

```diff
@@ -615,7 +614,9 @@ static Action<object> GetValidationActionFallback(Type type)
 
         // Get or compute the cached list of properties to validate. Here we're using a static lambda to ensure the
         // delegate is cached by the C# compiler, see the related issue at https://github.com/dotnet/roslyn/issues/5835.
-        EntityValidatorMap.GetValue(GetType(), static t => GetValidationAction(t))(this);
+        EntityValidatorMap.GetValue(
+            GetType(),
+            [RequiresUnreferencedCode("The type of the current instance cannot be statically discovered.")] static (t) => GetValidationAction(t))(this);
```

Comment:
You can now just write `GetValidationAction`. The linked Roslyn issue has been closed.

Reply:
The issue is closed but the fix is slated for the C# 11 timeslot, I don't think it's there in VS stable just yet, meaning our CI will be using an older version of Roslyn too without that. I would keep that as is for now (especially given it's unrelated from this PR anyway), and we can update this later on when we bump the SDK as well 🙂